lazy-scripts/scan.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh -l
+
+set -euo pipefail
+
+SONAR_ORGANIZATION=""
+SONAR_PROJECT_KEY=""
+REVISION=""
+BRANCH_NAME=""
+PULL_REQUEST_KEY=""
+
+function process_cli
```

Comment:
Tech debt ticket to make scripts consistant

Reply:
Ticket created to track in next sprint.

Reply:
Next time, please share the link to the ticket

Reply:
https://usxtech.atlassian.net/browse/LAZY-51